src/java.base/share/classes/sun/security/x509/NamedX509Key.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package sun.security.x509;
+
+import sun.security.util.BitArray;
+
+import java.io.IOException;
+import java.io.InvalidObjectException;
+import java.io.ObjectInputStream;
+import java.io.Serial;
+import java.security.InvalidKeyException;
+import java.security.KeyRep;
+import java.security.NoSuchAlgorithmException;
+import java.security.ProviderException;
+import java.security.spec.NamedParameterSpec;
+
+public final class NamedX509Key extends X509Key {
+    @Serial
+    private static final long serialVersionUID = 1L;
+
+    private final String fname;
+    private final transient NamedParameterSpec paramSpec;
+    private final byte[] h;
+
+    /// Ctor from family name, parameter set name, raw key bytes.
```

Comment:
Any reason you use 3 slashes instead of 2 for comments?

Reply:
I'm trying out the https://openjdk.org/jeps/467 style javadoc. Still, this is not public API so I can take the risk for the experiment.

Reply:
Ok, but how are you experimenting? These comments aren't using any markdown that I can see, and it's rare that we would generate javadoc for internal classes.

Reply:
I have quite a lot in `NamedKeyPairGenerator` and IntelliJ IDEA recognizes them if you press F1 inside.

Reply:
Not many markdown, I mainly want to practice link, code, params, return, throws, etc. `NamedKeyFactory` has a numbered list.